src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,54 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
+
+  getAnimalHealth(name) {
+    return Animal.alive.find((item) => item.name === name).health;
+  }
+
+  getBited() {
+    Animal.alive = Animal.alive.filter((item) => item.health !== 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+
+    Animal.alive.push(this);
+  }
+
+  bitten() {
+    this.health -= 50;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+
+    Animal.alive.push(this);
```

Comment:
This should happen in the constructor of an Animal - or you will have to duplicate it for each `Animal` subclass

Reply:
This is a problem for me.
Because I have no idea how to send hidden property from Herbivore to Animal.
Please advise)